workflow-style.qmd

```diff
@@ -1,239 +1,237 @@
-# Workflow: code style {#sec-workflow-style}
+# Fluxo de trabalho: convenção de estilo {#sec-workflow-style}
 
 ```{r}
 #| echo: false
 
 source("_common.R")
 ```
 
-Good coding style is like correct punctuation: you can manage without it, butitsuremakesthingseasiertoread.
-Even as a very new programmer, it's a good idea to work on your code style.
-Using a consistent style makes it easier for others (including future-you!) to read your work and is particularly important if you need to get help from someone else.
-This chapter will introduce the most important points of the [tidyverse style guide](https://style.tidyverse.org), which is used throughout this book.
+As boas práticas de convenção de estilo em programação são como as regras gramaticais: você pode até viver sem elas, mas elas certamente tornam asCoIsaSmaisfáCeiSdeler.
+Mesmo como um programador iniciante, é uma boa ideia desenvolver boas práticas na escrita do seu código.
+Usar um estilo de escrita consistente torna mais fácil para os outros (incluindo o seu eu do futuro!) ler o seu trabalho e é particularmente importante se você precisar de ajuda de outra pessoa.
+Este capítulo apresentará os pontos mais importantes do [guia de estilo tidyverse](https://style.tidyverse.org), que é usado em todos os capítulos deste livro.
 
-Styling your code will feel a bit tedious to start with, but if you practice it, it will soon become second nature.
-Additionally, there are some great tools to quickly restyle existing code, like the [**styler**](https://styler.r-lib.org) package by Lorenz Walthert.
-Once you've installed it with `install.packages("styler")`, an easy way to use it is via RStudio's **command palette**.
-The command palette lets you use any built-in RStudio command and many addins provided by packages.
-Open the palette by pressing Cmd/Ctrl + Shift + P, then type "styler" to see all the shortcuts offered by styler.
-@fig-styler shows the results.
+Formatar seu código de acordo com as convenções de estilo pode parecer um pouco chato no início, mas se você praticar, isso vai se tornar natural rapidamente.
+Além disso, existem algumas ótimas ferramentas para formatar de forma rápida códigos antigos, como o pacote [**styler**](https://styler.r-lib.org) de Lorenz Walthert.
+Depois de instalar o pacote com o comando `install.packages("styler")`, uma maneira fácil de usá-lo é através da paleta de comando (**command palette**) do RStudio.
```

Comment:
"paleta de comando" é meio estranho... mas fui tentar entender do que se trata e eu nem sabia que isso existia  🤡  (abri com Cmd/Ctrl + Shift + P) 

então não sei se tem alguma outra forma que as pessoas se referem à isso

Reply:
É o mesmo atalho do VSCode, tbm chama command palette lá hahaha, tbm n sabia que tinha no RStudio, vivendo e aprendendo.

Reply:
pois eu uso muito essa funcionalidade no VSCode! e sempre pensava "bem que o RStudio podia ter algo assim" ahahhahahaahuhau

Reply:
Acho que não tem outro nome não... talvez, caixa de entrada de comandos? e aí deixaria command pallet antre parêntesis.

Reply:
https://docs.github.com/pt/codespaces/codespaces-reference/using-the-vs-code-command-palette-in-codespaces vendo aqui, pro VSCode também traduzem como paleta de comando. então é isso, é esquisito mas é o que é usado! Obrigada